src/features/pilot_management/PilotSheet/components/VaultDialog.vue

```diff
@@ -84,6 +84,11 @@ export default Vue.extend({
     syncing: false,
     copyConfirm: false,
   }),
+  computed: {
+    getShareCode() {
+      return this.pilot.ShareCode
+    }
+  },
```

Comment:
Make the pilot ShareCode a computed property so it updates in a reactive manner.